### DISCUSSION TITLE
Telnet Server Using Textual

### DISCUSSION THREAD

**Discussion by shibukraj** [Textualize/textual#75] [Ideas]
Greetings there,

I am very new to Python and currently learning python. I am working on a project that is designed as a interactive terminal server. The current implementation is done using BlessedJS and Node-Telnet. You can see a similar example here - https://github.com/chjj/node-telnet2

My question here is , can you share your thoughts in using Textual in the context similar to above. If so do you have any ideas on any python lib we can use similar to telnet2. Any ideas, suggestions, thoughts is really appreciated.

**Comment by willmcgugan** [comment]
I'm not sure what that project does in the terminal, so I can only guess. I would expect Textual to be able match other libraries for features *eventually*, if not already.
